client/src/elm/Pages/Prenatal/ProgressReport/View.elm

```diff
@@ -2144,5 +2144,33 @@ viewTreatementForDiagnosis language date measurements allDiagnoses diagnosis =
             -- @todo
             []
 
+        DiagnosisPelvicPainIntense ->
+            --@todo
```

Comment:
Note the todo, in case it's needed here :)

Reply:
@nedSaf 
This is a placeholder for Progress report implementation.
Got a separate issue for it.